ansible/roles/optimizations/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+- name: Disable daily upgrades
+  systemd:
+    name: apt-daily-upgrade.timer
+    state: disabled
```

Comment:
missing i think some others services, see that : https://superuser.com/questions/1327884/how-to-disable-daily-upgrade-and-clean-on-ubuntu-16-04